tools/rowlog/rowlog.go

```diff
@@ -301,10 +301,12 @@ func processRowEvent(plan *TablePlan, gtid string, ev *binlogdatapb.VEvent) []*R
 			before = sqltypes.MakeRowTrusted(plan.fields, change.Before)
 		}
 		for _, val := range after {
-			afterVals = append(afterVals, string(val.ToBytes()))
+			bytes, _ := val.ToBytes()
```

Comment:
not good to ignore errors.
@rohit-nayak-ps should we log an error and move on, or fail?

Reply:
We could just panic , to exit the rowlog tool, in case of an errorhere , since we are just parsing binlogs and it should not normally occur. Of if it did we would have already seen failing vreplication workflows.

Reply:
Fixed via d15ec9eec8